DifferentLocation.java

```diff
@@ -0,0 +1,120 @@
+package location;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class DifferentLocation {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/";
+		driver.get(appUrl);
+
+		// Click on Location
+
+		driver.findElement(By.linkText("Location")).click();
+
+		List<WebElement> allElements = driver.findElements(By.xpath("//div[1]/div[3]/div/nav/ul/li[5]/ul"));
+
+		for (WebElement element : allElements) {
+			System.out.println(element.getText());

```

Comment:
Rest of the code should be inside this for loop not outside.
Use `element` variable itself to click.